bin/fin

```diff
@@ -2861,6 +2864,41 @@ ssh_add ()
 	return "$ret"
 }
 
+# Loop through and add defined ssh keys
+_add_ssh_keys_defined () {
```

Comment:
Let us not separate it out into a function. It's logically the same ssh-add operation and functionally will never be re-used outside of ssh-add